1a_workflowed/README.md

```diff
@@ -0,0 +1,188 @@
+
+In addition to the workflow script, the workflow run directory will contain all artifacts which are created as part of the run. This means that a CARB scientist can run successive, iterative versions of each workflow until the desired outcome is acheived. Each individual run is preserved in its entirety, and the scientist can always reference the specific workflow run which produced the desired outcome by its unique run identifier.
+
+By referencing the specific workflow run (by its identifier), an individual is also able to reference the specific data artifacts generated by that workflow.
+
+### Workflow re-evaluation
+One of the benefits of using a workflow framework is that we are enabled to leverage efficient workflow run re-evaluation. 
+
+This means that if a workflow is invoked with a run identifier that already exists, that invocation will only execute steps of the workflow if either the inputs to that step have changed, or if the code for that workflow step has changed.
+
+### Data Referencing
+As part of the workflow framework established, an individual is able to reference data external to a workflow by invoking specific workflow steps within the workflow definition. Specifically, the data artifacts of a particular run of Workflow A may be referenced by Workflow B, using the run identifier of the specific iteration of Workflow A desired. This allows the creation of modular, extensible workflows which depend on common data resources from earlier steps.
+
+Disciplined execution of workflows with attention paid to run identifiers will enable the creation of standardized validated data products suitable for use by a broad array of data scientists.
+
+See the **(HP: need to create a multipart workflow example)** for an example of creating a data handling workflow prior to an analytics workflow.
+
+### Distributed Compute for Workflows
+In order to execute the workflow in a distributed manner, individual workflow steps are invoked within the specific workflow run directory. To accomplish this, a new R process is instantiated on the worker node, and the code is executed as part of the Targets framework.
+
+This means that the workflow steps - as invoked by slurm within an R process - have access to the workflow run resources, such as data artifacts produced by preceding steps. It will also be executed in the context of the workflow run directory, and so the invocation of PEcAn methods within the workflow directory becomes quite direct. This should also make it clear that, as the step is invoked within a new R namespace (and indeed, on an entirely different compute node), each workflow step must import its own dependencies.
```

Comment:
It would be helpful to document why crew + crew.cluster packages are not used for distributed computing here.

Is this a documented limitation or is there an open bug report / feature request related to this?